### PR TITLE
Even more additions to the 1.13 Release Team

### PR DESCRIPTION
**Pull request by justaugustus**
/assign @AishSundar 
/sig release

Signed-off-by: Stephen Augustus <foo@agst.us>

### CODE REVIEW

File: releases/release-1.13/release_team.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Dhawal Yogesh Bhanushali ([@imkin](https://github.com/imkin)) |
-| Bug Triage | Niko Penteridis ([@nikopen](https://github.com/nikopen)) | Mohammed Zeeshan Ahmed ([@mohammedzee1000](https://github.com/mohammedzee1000)), Athanasios Kostopoulos ([@thanasisk](https://github.com/thanasisk)), Shubheksha Jalan ([@shubheksha](https://github.com/shubheksha)) |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Dhawal Yogesh Bhanushali ([@imkin](https://github.com/imkin)), Conrad Taylor ([@conradwt](https://github.com/conradwt)), Rahul Mahale ([@RahulMahale](https://github.com/RahulMahale)) |
+| Bug Triage | Niko Penteridis ([@nikopen](https://github.com/nikopen)) | Mohammed Zeeshan Ahmed ([@mohammedzee1000](https://github.com/mohammedzee1000)), Athanasios Kostopoulos ([@thanasisk](https://github.com/thanasisk)), Shubheksha Jalan ([@shubheksha](https://github.com/shubheksha)), Tunde ([@simplytunde](https://github.com/simplytunde)) |
```
Comment (AishSundar) [inline releases/release-1.13/release_team.md]: As per guidance in the SIG Release meeting, we decided to cap the shadows to 4 per role. So probably we can add Tunde as an official shadow for 1.14? But feel free to join the 1.13 meetings if you are interested @simplytunde

**Comment by AishSundar** [comment]
/lgtm
/hold

Please remove hold after addressing the comment above.Thanks @justaugustus for getting all this rounded up.

File: releases/release-1.13/release_team.md (hunk 1 above)
Comment (justaugustus) [reply releases/release-1.13/release_team.md]: Yep. It's capped at four here, including Tunde.

**Pull request merged**